connext_cmake_module/cmake/Modules/FindConnext.cmake

```diff
@@ -112,19 +112,9 @@ function(_find_connext_libraries var library_names basepath)
     # If this is Darwin, only accept Darwin
     if(APPLE)
       set(match FALSE)
-      string(FIND "${library}" "x64Darwin17clang9.0" _found)
-      if(NOT ${_found} EQUAL -1)
+      string(REGEX MATCH ".*x64Darwin[1-9][0-9]clang[0-9]+.[0-9]" _found "${library}")
```

Comment:
Without using `^` do you really need the leading `.*`?

Reply:
It changes the value stored in `_found`, but since we're just checking if it's not empty we don't really need the leading `.*`.

Reply:
Removed 18734ac